my-app/src/App.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import Header from './components/Header';
+import Navbar from './components/Navbar';
+import Profile from './components/Profile';
+
```

Comment:
Consider removing redundant new lines.

Reply:
I don't quite understand what you mean..?

Reply:
There're two empty new lines between "import Profile from './components/Profile';" and "import './App.css';"
Should be just one :)